<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="jlast-5659" ignored="4" name="Test Android" tests="4" failures="1" timestamp="2022-04-23T21:05:47 MDT" time="87.641" errors="0">
  <testcase name="slidingMenu" time="29.927" classname="com.qa.tests.SlidingMenuTests"/>
  <testcase name="dashboardTests" time="7.822" classname="com.qa.tests.DashboardTests"/>
  <testcase name="settingFailTest" time="2.874" classname="com.qa.tests.SettingsTests">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.qa.BaseTest.verifyText(BaseTest.java:124)
at com.qa.tests.SettingsTests.settingFailTest(SettingsTests.java:98)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- settingFailTest -->
  <testcase name="settingsTest" time="28.636" classname="com.qa.tests.SettingsTests"/>
  <testcase name="@AfterTest afterTest" time="0.0" classname="com.qa.BaseTest">
    <ignored/>
  </testcase> <!-- @AfterTest afterTest -->
  <testcase name="@AfterTest afterTest" time="0.0" classname="com.qa.BaseTest">
    <ignored/>
  </testcase> <!-- @AfterTest afterTest -->
  <testcase name="@BeforeTest beforeTest" time="0.0" classname="com.qa.BaseTest">
    <ignored/>
  </testcase> <!-- @BeforeTest beforeTest -->
  <testcase name="@BeforeTest beforeTest" time="0.0" classname="com.qa.BaseTest">
    <ignored/>
  </testcase> <!-- @BeforeTest beforeTest -->
</testsuite> <!-- Test Android -->
